StatEngine/stat-engine
# Unit Filtering

## Overview
Adds the ability for customers to filter units for email & unit reports.

## GitHub Issues
- https://app.asana.com/0/1177539460427774/1199114428166459

## Changes
- Filter page in settings
- Auto filtering units

## Screenshots / Videos
![image](https://user-images.githubusercontent.com/2092432/106208144-eb8cf300-6190-11eb-91f0-a82d0b5a5ea8.png)


## Steps to Test
- Open app
- Login
- Go to settings
- Apply filters
- Go to reports -> units
- You should not see the units you've filtered.
